edenai_apis/apis/leonardo/leonardo_api.py

```
@@ -145,3 +146,65 @@ def image__generation(
+        response_dict = await get_response_async()
+        generation_by_pk = response_dict.get("generations_by_pk", {}) or {}
+        generated_images = generation_by_pk.get("generated_images", []) or []
+
+        image_url = [image.get("url") for image in generated_images]
+
+        generated_images = []
+        if isinstance(image_url, list):
+            for image in image_url:
+                generated_images.append(
+                    GeneratedImageDataClass(
+                        image=base64.b64encode(requests.get(image).content),
```

Ruff (0.14.3)

Probable use of `requests` call without timeout

(S113)